src/controllers/action-queue/types/pushTypes.js

```diff
@@ -54,7 +54,7 @@ export type PushMessage = {
   +data?: { [key: string]: string } // JSON to push to device
 }
 
-export type PushEventState = 'waiting' | 'cancelled' | 'complete'
+export type PushEventState = 'waiting' | 'cancelled' | 'triggered'
```

Comment:
The "cancelled" and "hidden" states both mean the event will not be returned from the API. So you should either keep both (since they are valid states) or remove both (for consistency). Probably keep both, in case we update the API with options like `includeHidden` or `includeCancelled`.